fusor-ember-cli/app/controllers/assign-nodes.js

```diff
@@ -286,10 +272,8 @@ export default Ember.Controller.extend(
           "X-CSRF-Token": token
         }
       }).catch(function (error) {
-        error = error.jqXHR;
         console.log('ERROR');
         console.log(error);
-        self.set('showLoadingSpinner', false);
```

Comment:
it was left over from when this UI used to lock up everything and show a spinner any time there was any change.